Create get-an-international-relocation-payment.md (#3300)
DFE-Digital/get-into-teaching-app: app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

The international relocation payment (IRP) is designed to repay some of the costs of moving to England, including:

* the costs of visas

* the immigration health surcharge

* other relocation expenses

All eligible applicants will receive a single payment of £10,000 from the Department for Education (DfE) around the end of their first term training or working in England. The IRP is a grant – you will not have to pay this money back. You can only receive the IRP once.

## Criteria for trainee teachers  

As a trainee teacher, to be eligible for the international relocation payment (IRP), you must meet all 3 of the following requirements.

### 1. Teaching subject (trainees)

You must have taken up a place on a teacher training course in England leading to [qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts) for the academic year 2023 to 2024. Your teacher training provider must be accredited by the UK government, and you must be training to teach one of the following subjects at secondary level (teaching children aged 11 to 16 or 11 to 18):

> * any secondary level language(s) except English language, including courses combining language(s) with another subject, as long as language(s) make up at least 50% of the course content

Reviewer: because you've already said secondary level up above, I wonder if it's necessary for you to say it again in the bullets?

Author: yes I think you are right

### 1. Teaching subject (teachers)

You must have accepted a job as a teacher of languages or physics in an English state secondary school on a contract lasting at least one academic year.  

Teachers of all languages offered in English state secondary schools are eligible to apply for the IRP. Language(s), with the exception of English language, can be combined with another subject, but must make up at least 50% of teaching time. You can, however, combine teaching English language with another language (for example, French, German or Spanish), as long as the non-English language makes up at least 50% of your teaching timetable.  

Physics can be taught with another subject, but must make up at least 50% of teaching time. Teachers of general science or combined science  are also eligible to apply for the IRP, as long as they are teaching the physics element of general science or combined science. General science or combined science can be taught with another subject, but must make up at least 50% of teaching time.  

Check with your employing school if you’re not sure whether your teaching timetable meets the teaching subject requirements.

> 'State schools' are schools funded by the UK government. 'Secondary schools' teach children aged 11 to 16 or 11 to 18. You will not be eligible if you teach in a school for pupils aged 16 to 18 only.  

Reviewer: ```suggestion
State schools are schools funded by the UK government. Secondary schools teach children aged 11 to 16 or 11 to 18. You will not be eligible if you teach in a school for pupils aged 16 to 18 only.  
```
do we need quote marks here?

Author: Probs not

> An academic year is 3 full school terms: for example, if your teaching job started in September 2023 (the start of the autumn term), it would need to continue until at least July 2024 (covering the spring and summer terms). ‘Ongoing’ or ‘permanent’ contracts lasting longer than 1 academic year are also eligible.

Reviewer: ```suggestion
An academic year is 3 full school terms: for example, if your teaching job started in September 2023 (the start of the autumn term), it would need to continue until at least July 2024 (covering the spring and summer terms). Ongoing or permanent contracts lasting longer than 1 academic year are also eligible.
```
do we need quote marks here?

Author: Probs not

The Department for Education reserves the right to withhold payment if you are no longer studying on an eligible course or employed in an eligible job on the payment date.

Once you have received the international relocation payment, you will not be asked to pay it back, even if you leave England, your training course or your job as a teacher.

### Tax

If you are on a fee-paying teacher training course, you will not have to pay tax on the international relocation payment.  

If you are a teacher, or a trainee on a salaried course, the Department for Education will pay the UK tax owed on the international relocation payment to ensure you receive the full £10,000, as long as your overall earnings do not exceed £50,270 a year in 2023/24. You may have to pay higher rate [UK Income Tax](https://www.gov.uk/browse/tax/income-tax) and [UK National Insurance](https://www.gov.uk/national-insurance) on any earnings which take you over £50,270.

### Part-time teachers and trainees

Part-time teachers and trainees will receive the international relocation payment in full, as long as they meet all the eligibility criteria.

### Other types of financial support from the UK government  

> Teachers and newly qualified trainees who meet the eligibility requirements can also apply for the following payments in addition to the international relocation payment (IRP):

Reviewer: ```suggestion
Teachers and recently qualified trainees who meet the eligibility requirements can also apply for the following payments in addition to the international relocation payment (IRP):
```
just to move away from 'newly qualified teacher' as we don't say this any more?

Author: good point

---

> title: "Get an international relocation payment (IRP) worth £10,000"

Author: Yes that's better

description: |-

>   Non-UK teachers and trainees in physics and languages could be eligible for financial help from the UK government.

Reviewer: makes sense to include 'international relocation payment' in the description to help with search, if this is something users hear about and search for in Google?

Author: Good idea

    Teach in England if you trained outside the UK: "/non-uk-teachers/teach-in-england-if-you-trained-overseas"
    Apply for your visa to teach in England: "/non-uk-teachers/visas-for-non-uk-teachers"
    Apply for your visa to train to teach in England: "/non-uk-teachers/fees-and-funding-for-non-uk-trainees"

promo_content:
date: "2021-05-27"
image: "static/content/hero-images/0002.jpg"
backlink: "../../"
keywords:
  - International
  - Overseas
  - Foreign
  - Nationality
  - Residency
  - EEA
  - Canada
  - New Zealand
  - Switzerland
  - USA
  - Visa
  - Skilled worker visa
  - EU
  - EU Settlement Scheme
  - NARIC
  - UK NARIC
  - ENIC
  - UK ENIC
  - European Network of Information Centre
  - National Recognition Information Centre for the United Kingdom
  - PhD
  - Immigration
  - Ireland
  - Irish
  - Common Travel Area
  - Student Visa
  - Home Office
  - Graduate Visa
  - Youth Mobility Scheme
  - Japan
  - Monaco
  - Hong Kong
  - Republic of Korea
  - Taiwan
  - Family Visa
  - UK Ancestry Visa
  - Abroad
  - Europe
  - European
  - Settlement Scheme
---

> ## Who is eligible

Author: I would like to keep unless you fiercely object :)

The information on this page is for: 

* non-UK teachers of languages and physics 
* non-UK trainee teachers of languages and physics   

If you are considering coming to England to teach or train to teach languages or physics, you could be eligible for financial help from the UK government worth £10,000.  

The international relocation payment (IRP) is designed to repay some of the costs of moving to England, including: 

* the costs of visas 

* the immigration health surcharge 

* other relocation expenses 

If you are eligible, you’ll receive a single payment of £10,000 from the Department for Education (DfE) around the end of your first term training or working in England. You will not have to pay this money back. You can only receive the IRP once. 

> ## How to apply 

Reviewer: this heading/section to me seems a bit superfluous, if you've got the information further down about how both trainees and teachers can receive the payment?

The process of applying for the international relocation payment is different for trainee teachers and teachers.  
 
## Criteria for trainee teachers  

As a trainee teacher, to be eligible for the international relocation payment (IRP), you must meet all 3 of the following requirements.

### 1. Teaching subject
 
You must have taken up a place on a teacher training course in England leading to [qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts) for the academic year 2023 to 2024. Your teacher training provider must be accredited by the UK government, and you must be training to teach one of the following: 

> a) any secondary language or languages, including courses with content which is at least 50% languages. You can combine training to teach English with another language, as long as the non-English language makes up at least 50% of your course. 

Reviewer 2: This needs to be coded as a list from an accessibility point of view

Author: Ah really OK cool

> a) any secondary language or languages, including courses with content which is at least 50% languages. You can combine training to teach English with another language, as long as the non-English language makes up at least 50% of your course. 

Author: I know lols that's why I've made them full sentences not bullets. Is it better to have an immensely long sentence to comply with GOV.UK style guide on bullets, or avoid bullets and use another kind of formatting so as to have shorter and more comprehensible sentences...?

> a) any secondary language or languages, including courses with content which is at least 50% languages. You can combine training to teach English with another language, as long as the non-English language makes up at least 50% of your course. 

Author: I broke this up into 3 bullets in the end

> b) secondary physics, including [Engineers teach physics](/subjects/engineers-teach-physics), and courses with content which is at least 50% physics. 

Author: done

> You can search for [postgraduate teacher training courses](https://www.gov.uk/find-postgraduate-teacher-training-courses) accredited by the UK government leading to the award of QTS. You will still be eligible for the international relocation payment if your course leads to PGCE + QTS or PGDE + QTS. 

Author: done

### Fee-paying trainees 

If you have a place on a teacher training course where you pay a fee, you do not need to apply for the international relocation payment (IRP). Your training provider will email you if your choice of course makes you eligible for the IRP. Your visa status and date of arrival will be checked by the Department for Education. If you are eligible, you’ll get the IRP around the end of your first term. 

If you believe you meet the eligibility requirements, but have not been informed by your teacher training provider that you may be awarded the IRP, please contact your training provider directly first. If your training provider cannot help you, you can email us at teach.inengland@education.gov.uk. 

### Salaried trainees 

If you have a place on a teacher training course where you are paid a salary to work while you train, email us at IRP.ExpressInterest@education.gov.uk to let us know you are interested in applying for the IRP. You will be asked to fill in a form giving us your details and we’ll get in touch when IRP applications open for salaried trainees in autumn 2023. If you are eligible, you’ll get the IRP around the end of your first term. 

> ## The international relocation payment for teachers  

Author: Done

To be eligible for the international relocation payment (IRP), you must meet all 3 of the following requirements.

> ### 1. Teaching subject

Reviewer: Is it good practice to have 2 headings with the same text? (I'm thinking this is probably bad for accessibility, and not useful for users trying to scan through to find the relevant information for them)

Reviewer 3: yes - it is not good practice in terms of accessibility to have duplicate headings on a single page

Author: I have relabelled them...

You must have accepted a job as a teacher of languages or physics in an English state secondary school on a contract lasting at least one academic year.  

> a) Teachers of all languages offered in English state secondary schools are eligible to apply for the IRP. The language or languages can be combined with another subject, but must make up at least 50% of teaching time. You can combine teaching English with another language, as long as the non-English language makes up at least 50% of your teaching timetable.  

Reviewer 2: This needs to be coded as a list from an accessibility point of view

> a) Teachers of all languages offered in English state secondary schools are eligible to apply for the IRP. The language or languages can be combined with another subject, but must make up at least 50% of teaching time. You can combine teaching English with another language, as long as the non-English language makes up at least 50% of your teaching timetable.  

Author: I've changed these to paragraphs as they're too complicated to write as bullets...

b) Physics can be taught with another subject, but must make up at least 50% of teaching time. Teachers of general science or combined science  are also eligible to apply for the IRP, as long as they are teaching the physics element of general science or combined science. General science or combined science can be taught with another subject, but must make up at least 50% of teaching time.  

> c) 'State schools' are schools funded by the UK government. 'Secondary schools' teach children aged 11 to 16 or 11 to 18. You will not be eligible if you teach in a school for pupils aged 16 to 18 only.  

Reviewer: would this information be better served where you first mention state and secondary schools? This info could go in brackets after the 'You must have accepted a job ...' sentence

Author: I have restructured this section...

d) An academic year is 3 full school terms: for example, if your teaching job started in September 2023 (the start of the autumn term), it would need to continue until July 2024 (covering the spring and summer terms). 

[Check the type and age range of the school](https://www.get-information-schools.service.gov.uk/) where you’ll be teaching. You can also find out about the [different types of state school in England](https://www.gov.uk/types-of-school). 

> ### 2. Type of visa

Reviewer: Is it good practice to have 2 headings with the same text? (I'm thinking this is probably bad for accessibility, and not useful for users trying to scan through to find the relevant information for them)

Author: Relabelled

You must come to England on one of the visas listed here.  
 
For the following visas, you must have applied for your visa from outside the UK:  

> * [Skilled worker visa](https://www.gov.uk/skilled-worker-visa) 

Reviewer: given that many of these visas are the same for teachers/trainees, would it not make sense to combine these two sections and just have one section on visas? then if you need to clarify any, for example, the student visa, you could just put extra detail after the visa in brackets, like so:

'Student visa (if you're training to teach in England)'

* [Youth Mobility Scheme](https://www.gov.uk/youth-mobility)  

* [India Young Professionals Scheme visa](https://www.gov.uk/india-young-professionals-scheme-visa) 

* [Family visa](https://www.gov.uk/uk-family-visa) 

* [UK Ancestry visa](https://www.gov.uk/ancestry-visa)  

* [British National (Overseas) visa](https://www.gov.uk/british-national-overseas-bno-visa) 

* [High Potential Individual visa](https://www.gov.uk/high-potential-individual-visa)  

For the following Ukrainian and Afghan visas, you can apply for your visa from within the UK:  

* [Afghan citizens resettlement scheme](https://www.gov.uk/guidance/afghan-citizens-resettlement-scheme) 

* [Afghan Relocations and Assistance Policy](https://www.gov.uk/government/publications/afghan-relocations-and-assistance-policy/afghan-relocations-and-assistance-policy-information-and-guidance) 

* [Ukraine Family Scheme visa](https://www.gov.uk/guidance/apply-for-a-ukraine-family-scheme-visa) 

* [Ukraine Sponsorship Scheme](https://www.gov.uk/guidance/apply-for-a-visa-under-the-ukraine-sponsorship-scheme) 

> ### 3. Date of arrival 

Reviewer: again, probably not good practice to have 2 headings with the same text, and given the info provided is the same for trainees AND teachers, could we not just have one section for both?

Author: Don't want to combine sections because the process and eligibility is different for teachers and trainees and I think it'll confuse things to blend them together. But I can relabel heads.

You must move to England no more than 3 months before the start of your job.  

> ### Non-eligibility for teachers 

Author: I really want to keep the sections separate for teachers and trainees – it's complicated enough as it is...

You will not be eligible for the international relocation payment if you are: 

* a UK citizen 

* an Irish citizen 

* a non-UK citizen who moved to England to live permanently more than 3 months before the start of your job 

* teaching only English language and literature

* employed on a contract lasting less than one academic year 

* employed by a recruitment agency 

## How teachers can apply for the international relocation payment 

You’ll need to have started your teaching job in a state secondary school to make your application for the international relocation payment (IRP). [Learn more about finding a teaching job and applying for your visa](/non-uk-teachers/teach-in-england-if-you-trained-overseas). 

You will be able to apply for the IRP from 1 September to 31 October 2023. You can email us at IRP.ExpressInterest@education.gov.uk to let us know you are interested in applying for the IRP. You will be asked to fill in a form giving us your details and we’ll get in touch when IRP applications open. If you are eligible, you’ll receive the money around the end of your first school term of employment. 

> For information about IRP applications in 2024, check back to this page. 

Author: done

## General information about the international relocation payment  

> Further information for trainees and teachers

Author: Thought h2s needed a line of text underneath before the the h3 (they do on GOV.UK) but happy to delete.

### Protect yourself from fraud 

Teachers and salaried trainee teachers will not need to pay an agent or any other third party to complete their application. Any agency or person contacting you offering to apply for the IRP on your behalf is likely to be fraudulent. [Avoid and report internet scams and phishing](https://www.gov.uk/report-suspicious-emails-websites-phishing). 

### Visiting England more than 3 months before the start of your course or job 

Your eligibility for the international relocation payment (IRP) will not be affected if you spend short periods in England at a time more than 3 months before the start of your course or job. 

> Example 1: You attend a job interview in England under a Standard Visitor visa in May 2023. You return home, and learn you have been given the job, which starts in September 2023. You apply for a Skilled worker visa in June 2023, 3 months before the start of your job. You move to England in July 2023. You will still be eligible for the IRP. 

Author: The UR indicates people find this really confusing and want examples...

Example 2: You have a UK Ancestry visa and use it to visit relations in England in February 2023. You return home and successfully apply for a place on a teacher training course starting in September 2023. You move to England in June 2023. You will still be eligible for the IRP. 

> You will also still be eligible for the IRP if you live in another part of the UK (for example, Scotland or Wales) as long as you are employed in a school in England.  

Reviewer: ```suggestion
You will also still be eligible for the IRP if you're a non-UK citizen living in another part of the UK (for example, Scotland or Wales) as long as you are employed in a school in England.  
```
Do we need a bit of clarification here, so that Scottish/Welsh citizens don't think they're eligible for the IRP? (Unless they are?)

Author: Yes good point

If you are in any doubt about meeting the 3-month requirement, please email us for advice before planning your move. 

### Subject eligibility  

> The international relocation payment (IRP) has been limited to languages and physics because non-UK teachers play an important part in teaching these priority subjects. The Department for Education is piloting the IRP to encourage more non-UK teachers and trainees to bring their skills, experience and potential to English schools. 

Author: I think it's helpful for them to know why only physics and languages are eligible...

### Payment 

Eligibility requirement checks can only be completed after you’ve arrived in England. Therefore, the international relocation payment (IRP) cannot be made before you take up your teacher training place or teaching job. 

In order to be paid the IRP, you must have a UK bank account (this can be digital). If you are a teacher or a salaried teacher trainee, you must also have a [UK National Insurance number](https://www.gov.uk/apply-national-insurance-number). You can apply for this once you arrive in England, as long as your visa gives you the right to work and you have a confirmed job.  

You will receive a single payment from the Department for Education around the end of your first term of training or employment. You will be paid as and when your eligibility requirements have been checked – we cannot guarantee an exact payment date.  

You will not have to justify any expenses or give us receipts or any other evidence of the relocation costs you have incurred.  

The Department for Education reserves the right to withhold payment if you are no longer studying on an eligible course or employed in an eligible job on the payment date. 

### Tax 

> The Department for Education will pay the UK tax owed on the international relocation payment to ensure you receive the full £10,000, as long as your overall earnings do not exceed £50,270 a year in 2023/24. You may have to pay higher rate [UK Income Tax](https://www.gov.uk/browse/tax/income-tax) and [UK National Insurance](https://www.gov.uk/national-insurance) on any earnings which take you over £50,270. 

Reviewer: ```suggestion
You will receive the full £10,000, as long as your overall earnings do not exceed £50,270 a year in 2023/24. You may have to pay higher rate [UK Income Tax](https://www.gov.uk/browse/tax/income-tax) and [UK National Insurance](https://www.gov.uk/national-insurance) on any earnings which take you over £50,270. 
```
Could this be simplified, maybe like this? (If this is right?)

Author: I'm not sure a shorter version is clearer in this case because I think you need to understand you're being presented info about tax. The shorter version sounds like DfE will mysteriously penalise you for earning more than £50,270.

> ### Part-time teachers and trainees 

Reviewer: I'm wondering if this bit needs to be called out into its own section, or could be integrated into the other content? (But if you think it's important enough to pull out, then maybe it does!)

Author: I think helpful for part-timers to be able to scan content for their bit.

Part-time teachers and trainees will receive the international relocation payment in full, as long as they meet all the eligibility criteria. 

> ### Leaving England, your training course or job 

Author: Good idea

Once you have received the international relocation payment, you will not be asked to pay it back, even if you leave England, your training course or your job as a teacher.  

> ### Other types of financial support from the UK government  

Author: Erm, this reads weirdly to me

> ### Other types of financial support from the UK government  

Reviewer 2: Fair enough

Teachers and newly qualified trainees who meet the eligibility requirements can also apply for the following payments in addition to the international relocation payment (IRP): 

* [the levelling up premium payment](https://www.gov.uk/guidance/levelling-up-premium-payments-for-teachers)  
* [early career payments](https://www.gov.uk/guidance/early-career-payments-guidance-for-teachers-and-schools)  

However, you should be aware that you can only receive the IRP once, and you cannot receive both the levelling up premium payment and the early career payment in the same year. 

This means that teachers in their first year of employment in England could receive either the IRP and the levelling up premium payment, or the IRP and the early career payment. 

Trainee teachers could receive the IRP in their training year in England, and either the levelling up premium payment, or the early career payment in their first year of work as a teacher.  

> ## Information for teacher training providers 

Author: Agree that GiT isn't usually aimed at schools and providers but there is a precedent for this (on the Ukraine page). As it's a new policy I think justified to point providers at the guidance which is relevant for them.

> ## Information for teacher training providers 

Reviewer 3: @laurahermionetennant92 I agree with Phil - I don't think having provider information on GIT is right; it is taking us ever further from GIT's purpose and our core audience

 I think we should probably revisit the Ukraine provider information - that was really a temporary solution when the page was created?

> ## Information for teacher training providers 

Author: Hi @gemmadallmandfe – OK will remove :)

> Full details of how providers will receive funding to pay the international relocation payment can be found in the [‘Initial teacher training (ITT) bursary: funding manual’](https://www.gov.uk/government/publications/initial-teacher-training-itt-bursary-funding-manual). 

Author: This has been cut...

## Contact us 

> Please don’t send us your CV or enquiries about training places or teaching jobs, as the Department for Education does not directly run training courses or employ teachers. 

Author: done

> [Non-UK teachers](/non-uk-teachers) gives more information about training to teach or working as a teacher in England. 

Author: done

> For general enquiries about the IRP, please email us at teach.inengland@education.gov.uk. 

Author: Done
